auto_install/install.sh

```diff
@@ -2450,6 +2598,17 @@ restartServices(){
 				$SUDO systemctl restart wg-quick@wg0.service
 			fi
 		;;
+		Alpine)
+			if [ "${VPN}" == 'openvpn' ]; then
+				"${SUDO}" rc-update add openvpn default &> /dev/null
+				"${SUDO}" rc-service -s openvpn restart
+				"${SUDO}" rc-service -N openvpn start
```

Comment:
```
openvpn:x:101:102:openvpn:/dev/null:/sbin/nologin
::: Install Complete...
::: Restarting services...
 * Caching service dependencies ...                                                                                                                                                      [ ok ]
 * openvpn: `/etc/openvpn/openvpn.conf' is not readable
 * ERROR: openvpn failed to start
```

OpenVPN configuration lives at `/etc/openvpn/openvpn.conf`, or change the deamon to look for `server.conf`.